repo/packages/C/cassandra/0/config.json

```diff
@@ -79,19 +79,19 @@
       "description": "CPU shares to allocate to each Cassandra Server Instance.",
       "type": "number",
       "minimum": 0.0,
-      "default": 2.0
+      "default": 0.1
```

Comment:
I suspect the average EA cluster will not be doing a lot CPU-wise. Let's test this on RC1 and see what happens.